src/applications/planning-guidance/8832/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can't recall right now, but do we need to add anything to registry.json for this application?

Reply:
We will at some point, this is just a ticket to get the folder and file structure set up, nothing more.